README.md

```diff
@@ -40,12 +38,38 @@ export CMAKE_C_COMPILER=/usr/bin/gcc
 export CMAKE_CXX_COMPILER=/usr/bin/g++
 ```
 
-## As a submodule/external project
-If your project has CMake support, simply add this repo as a subdirectory:
+## Build debian dev package
+```
+cmake --build build --target package
+
+# Generates umd-dev-x.y.z-Linux.deb
+```
+
+# Integration
+UMD can be consumed by downstream projects in multiple ways.
+
+## From Source (CMake)
+You can link `libdevice.so` by linking against the `umd::device` target.
+
+### Using CPM Package Manager
+```
+CPMAddPackage(
+  NAME umd
+  GITHUB_REPOSITORY tenstorrent/tt-umd
+  GIT_TAG blozano/cmake_magic_0
```

Comment:
Leaving comment just as a note that we probably don't want to keep GIT_TAG to this branch when we merge it

Reply:
Good point. I will change it now.